collections/Lat_th/MS_Lat_th_c_20.xml

```diff
@@ -81,35 +70,40 @@
                         <layout columns="1" writtenLines="33">33 long lines</layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="caroline">Caroline minuscule</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="rubrication">The Gospel passages sometimes rubricated.</decoNote>
+                  </decoDesc>
                </physDesc>
                <history>
                   <origin>
-                     <origDate calendar="Gregorian" notAfter="1000" notBefore="0900">10th century</origDate>
-                     <origPlace>
-                        <country key="place_7000084">German</country>
-                     </origPlace>
-                  </origin>
-                  <provenance>Foliated in modern pencil 62, 63</provenance>
+                     <origDate calendar="Gregorian" notAfter="0900" notBefore="0875">9th century, 3rd quarter</origDate>
+                     <origPlace cert="medium">
+                        <country key="place_1000070">Eastern France</country>
+                     </origPlace> (?): Bernard Bischoff, <title>Katalog der festländischen Handschriften des neunten Jahrhunderts</title> (1998), no. 3811.</origin>
                   <provenance>Once used as covers of a binding</provenance>
-                  <provenance>J. Tregaskis, cat. 850, Jan. 1922, item 23</provenance>
+                  <provenance>Second leaf foliated in modern pencil <q>63</q>.</provenance>
+                  <provenance><orgName key="org_132262613">J. Tregaskis</orgName>, cat. 850 (Jan. 1922), item 23</provenance>
                   <provenance>
-                     <persName role="fmo" key="person_178"><!-- identification uncertain -->C. L. Ricketts,
-                     Chicago</persName>
-                  </provenance>
-                  <provenance>Tregaskis, cat. 990, Jan. 1931, item 1, and cat. 1013, 1935, item 1;
-                     given by <persName role="fmo" key="person_311"><!-- not found -->F. E. Norris</persName>, 1952.</provenance>
+                     <persName role="fmo" key="person_178">Coella Lindsay Ricketts</persName>, Chicago </provenance>
+                  <provenance><orgName key="org_132262613">Tregaskis</orgName>, cat. 990 (Jan. 1931), item 1; and cat. 1013 (1935), item 1</provenance>
+                  <acquisition>Given by <persName role="fmo" key="person_311">Francis Edward Boshear Norris</persName>, 1952.</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>Typescript description by Bodleian Library staff</source>
+                        <source>Revised from typescript description by Bodleian Library staff</source>
```

Comment:
Could this be clarified? Perhaps something like "Typescript description (n.d.) by anonymous Bodleian Library staff, revised (Dec. 2020) by Andrew Dunning with reference to the manuscript and to secondary literature as cited"?